instrumentation/oshi/library/src/main/java/io/opentelemetry/instrumentation/oshi/SystemMetrics.java

```diff
@@ -19,7 +19,9 @@
 
 /** System Metrics Utility. */
 public class SystemMetrics {
-  private static final String TYPE_LABEL_KEY = "type";
+  private static final String DEVICE_LABEL_KEY = "device";
```

Comment:
Shouldn't we do something like `private static final Labels STATE_USED = Labels.of(...)`? That way we don't have to create a labels object every time we record a value (at least I think that's what's happening right now)

Reply:
but it will not work for `Labels.of(DEVICE_LABEL_KEY, device, DIRECTION_LABEL_KEY, "transmit")`

Reply:
updated for state labels